client/asset/btc/btc.go

```diff
@@ -761,9 +762,47 @@ func (s *swapOptions) feeBump() (float64, error) {
 	return bump, nil
 }
 
+// fundMultiOptions are the possible order options when calling FundMultiOrder.
 type fundMultiOptions struct {
-	Split       *bool   `ini:"swapsplit"`
-	SplitBuffer *uint64 `ini:"splitbuffer"`
+	// Split, if true, and multi-order cannot be funded with the existing UTXOs
+	// in the wallet without going over the maxLock limit, a split transaction
+	// will be created with one output per order.
+	//
+	// Use the splitKey const defined above in the options map to set this option.
+	Split *bool
+	// SplitBuffer, if set, will instruct the wallet to add a buffer onto each
+	// output of the multi-order split transaction (if the split is needed).
+	// SplitBuffer is defined as a percentage of the output. If a .1 BTC output
+	// is required for an order and SplitBuffer is set to 5, a .105 BTC output
+	// will be created.
+	//
+	// Use the splitBufferKey const defined above in the options map to set this.
+	SplitBuffer *uint64
```

Comment:
This still doesn't quite explain the motivation for the split buffer.  As far as I am aware, it makes sense in the following situations:
- this is the quote asset in a market, not the base asset
- you might cancel the order later, and put up a new one with a slightly different rate, thus requiring a different quote amount for the same base assets amount e.g. still 1 lot

The above makes the most sense if the price is going down, thus requiring more quote asset for the same amount of base asset.  If price goes up, this would worsen the overlock.

Reply:
Yes that motivation is right. I'll add the motivation to the docs as well.

Reply:
Why would you want that? With the base asset, once you have an output exactly the size of the order you are placing, the best effort funding should match that output with the order every time it is cancelled and replaced.

Reply:
I'm not following. I'm saying that if market ABC-XYZ existed on server 1 with lot size 1 and fee rate 10, and market ABC-XYZ existed on server 2 with lot size 1 and fee rate 11, the sized ABC utxos from orders on server 1 would not be reusable for orders on server 2, and could result in split txs being generated. I don't consider this situation important, and mesh may make it impossible.

A more likely example might be two markets on the same server with the same base asset, one with lot size 10 and another with lot size 20. If we have a utxo sized at 20 with fees for one swap, it couldn't be used on the market with lot size 10 for a 2-lot order. Buffer fixes that.

Reply:
That's true, but I was imagining that generally all the orders for each market maker would pretty much always be on the books. Then, occasionally, there would be a cancel order and a new order in the next epoch in which likely the same output that was previously used would be used again.